user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+in more details in the :ref:`corresponding section <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+This capability can be utilized by the organization-level managers and HRs or other employees authorized to manage the 
+organization-wide calendars
+
+Users can view organization-wide calendars, add them to their own calendar views and copy related events to their own 
+calendars, regardless of this capability.
+
+
+.. _admin-capabilities-sys-calendar-events: 
+
+Manage system calendar events
+-----------------------------
+
+The capability enables users to create, edit and delete events in the system-wide calendars, which are described 
+in more details in the :ref:`corresponding section <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+This capability can be utilized by the company managers and HRs or other employees authorized to manage calendars at the 
+system-wide level.
+
+Users can view organization-wide calendars, add them to their own calendar views and copy related events to their own 
+calendars, regardless of this capability.
+
+
+.. _admin-capabilities-sys-calendars: 
+
+Manage system calendars
+-----------------------
+
+The capability enables users to :ref:`create <user-guide-calendars-system>` and 
+:ref:`manage <user-guide-calendars-manage>` system-wide calendars.
+
+It is usually available for system administrators and top managers.
+
+
+.. _admin-capabilities-passwords:
+ 
+Manage users' passwords
+-----------------------
+
+The capability enables users to change passwords of other users. Usually, it is only granted to the system 
+administrators, and can be used when :ref:`creating or editing a user record <user-management-users>`. 
+
+The capability does not influence the user's ability to 
+:ref:`edit own password <user-guide-getting-started-profile-password>` from the *"My User"* page.
+
+
+.. _admin-capabilities-merge:
+
+Merge entities
+--------------
+
+The capability enables users to :ref:`merge <user-guide-ui-components-grids-delete-merge>` several records of the same 
+entity. This is mostly applicable to the merge of :ref:`accounts <user-guide-accounts>`.
+
+The capability is usually used by the sales representatives, marketing associates, and other employees working with 
+accounts.
+
+
+.. _admin-capabilities-outlook:
+
+Outlook integration
+-------------------
+
+The capability is available for enterprise users only. If this capability is set to *"System"*, users with the roles 
+will be able to :ref:`download <user-guide-getting-started-my-user-outlook>` the latest version 
+of OroCRM add-in for Outlook. 
+
+If :ref:`OroCRM is synchronized with Outlook <user-guide-synch-outlook>`, you can use the add-in to synchronize 
+contacts, tasks, and calendar events between OroCRM and your Outlook. You can associate emails to accounts, contacts, 
+opportunities, and cases and create leads, opportunities and cases in OroCRM based on emails from the Outlook side bar.
```

Comment:
Edit: If :ref:`OroCRM is synchronized with Outlook <user-guide-synch-outlook>`, you can use the add-in to synchronize contacts, tasks, and calendar events between OroCRM and your Outlook account. You can also associate emails to accounts, contacts, opportunities, and cases, as well as create leads, opportunities, and cases in OroCRM based on emails from the Outlook sidebar.